lstchain/reco/dl0_to_dl1.py

```diff
@@ -35,7 +35,7 @@
 ### PARAMETERS - TODO: use a yaml config file
 
 
-allowed_tels = {1}  # select LST1 only
+allowed_tels = {1, 2, 3, 4}  # select LST1 only
```

Comment:
I take the opportunity of seeing to discuss the correctness of taking all LSTs and treat them as mono to increase statistics.
(I personally think there is no fundamental issue but one needs to be extra careful in the reconstruction, I have had problems in the past...)

Reply:
Yes, I also wanted to have this discussion. What I've been doing is to use 4 telescopes for training but only LST1 for testing the performance. Anyway, in dl0_to_dl1 (before training) I wanted to store the information of 4 telescopes in order to do future stereo checks that we discussed with @moralejo in Montpellier.